packages/language-server/src/server.ts

```diff
@@ -211,3 +250,26 @@ connection.listen();
 function isXMLView(uri: string): boolean {
   return /(view|fragment)\.xml$/.test(uri);
 }
+
+function executeCommand(
+  textDocument: TextDocument,
+  params: ExecuteCommandParams
+): void {
+  switch (params.command) {
+    case QUICK_FIX_STABLE_ID_COMMAND: {
+      const change = executeQuickFixStableIdCommand({
+        textDocument,
+        // @ts-expect-error - we already checked arguments exist
```

Comment:
instead of many (with many commands) `ts-expect-error` comments
consider treating the params.argument as `any` type, e.g:

```
  params: ExecuteCommandParams & { arguments: any}

```